com.zenorogue.HyperRogue.json

```diff
@@ -0,0 +1,58 @@
+{
+  "app-id":"com.zenorogue.HyperRogue",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"20.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command":"hyperrogue",
+  "finish-args":[
+    "--device=all",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=fallback-x11"
+  ],
+  "modules":[
+    "shared-modules/glew/glew.json",
+    "shared-modules/glu/glu-9.json",
+    "shared-modules/SDL/SDL-1.2.15.json",
+    "shared-modules/SDL/SDL_mixer-1.2.12.json",
+    "shared-modules/SDL/SDL_ttf-2.0.11.json",
+    {
+      "name":"SDL_gfx",
+      "sources":[
+        {
+          "type":"archive",
+          "url":"https://downloads.sourceforge.net/project/sdlgfx/SDL_gfx-2.0.25.tar.gz",
+          "sha256":"556eedc06b6cf29eb495b6d27f2dcc51bf909ad82389ba2fa7bdc4dec89059c0"
+        }
+      ]
+    },
+    {
+      "name":"hyperrogue",
+      "buildsystem":"simple",
+      "build-commands":[
+        "sed -i 's|TTF_OpenFont.*;|TTF_OpenFont(\"/usr/share/fonts/dejavu/DejaVuSans-Bold.ttf\", siz);|' basegraph.cpp",
+        "make",
+        "install -Dm755 hyperrogue -t ${FLATPAK_DEST}/bin/",
+        "install -Dm644 hyperroid/app/src/main/res/drawable-xxxhdpi/icon.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/com.zenorogue.HyperRogue.png",
+        "install -Dm644 com.zenorogue.HyperRogue.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/",
+        "install -Dm644 com.zenorogue.HyperRogue.desktop -t ${FLATPAK_DEST}/share/applications/"
+      ],
+      "sources":[
+        {
+          "type":"git",
+          "url":"https://github.com/zenorogue/hyperrogue.git",
+          "commit":"7fff0405a7806b5709fd311f15ef0b3d9a392450"
```

Comment:
is there a reason to use a snapshot rather than the "v12.0f" release tag? 

Which also lead to why 12.0c in the metainfo.

Reply:
Couldn't get the later ones to compile.

Reply:
you can be sure that users will complain about it not being the latest.